documentation/SDKs/javascript/README.md

```diff
@@ -7,352 +7,264 @@
 ### Installing the JavaScript SDK for Nether
 
 ## Usage
-
-### Analytics
-
-#### Active sessions - daily
-You can use the ***Nether.Analytics.GetActiveSessionsDaily*** method to get the total active sessions per day. Returns an array of ActiveSession
-##### Example usage
-```javascript
-var sessions = Nether.Analytics.GetActiveSessionsDaily();
-console.log(sessions[0].ActiveSessions + ' for ' + sessions[0].Day + '-' + sessions[0].Month + '-' + sessions[0].Year);
-```
-#### Active sessions - monthly
-You can use the ***Nether.Analytics.GetActiveSessionsMonthly*** method to get the total active sessions per month. Returns an array of ActiveSession
-##### Example usage
-```javascript
-var sessions = Nether.Analytics.GetActiveSessionsMonthly();
-console.log(sessions[0].ActiveSessions + ' for ' + sessions[0].Month + '-' + sessions[0].Year);
+To use the nether SDK include the nether script in your project. Reference the script in the webpage using 
+```html
+<script src="/scripts/nether.js"></script>
+```
+```javascript
+var config = {
+        netherClientId: '<client Id>',
+        netherClientSecret: '<client secret>',
+        facebookAppId: '<facebook app Id>',
+        netherBaseUrl: '<nether url>'
+    }
+    nether.init(config, facebookInitailised, netherInitialised);
```

Comment:
We're currently heavily focused on facebook auth, but the underlying APIs/design allows for multiple auth providers.

I'd like to see two authentication methods on the SDK at this point.
 1. loginWithNetherToken
 1. loginWithFacebook

Log in with Nether Token would just take a Nether access token, however it was obtained and use it. (This provides a hook in to use the JS SDK with other auth methods even if they're not directly incorporated into the JS SDK)

Log in with facebook would be the current method, so I'd see the facebookInitialised callback being passed in on that method. (And I expect that this would build on the previous method)

As an additional note, it would be fine to send analytics events from the client before being authenticated :-)